boards/arm64/xenvm/xenvm.dts

```diff
@@ -78,7 +78,8 @@
 	};
 
 	xen_hvc: hvc {
-		compatible = "xen,uart_hvc";
+		compatible = "xen,hvc-uart";
 		status = "okay";
+		label = "UART_HVC";
```

Comment:
Is this used? We are trying to move away from labels.

Reply:
There is a dependency loop I think. https://github.com/zephyrproject-rtos/zephyr/blob/1578dc699be8d334ec36123996922fef375bde81/dts/bindings/serial/uart-controller.yaml#L17 needs the label and the uart bindings includes `uart-controller.yaml`, so ...

But the @galak's cleanup work https://github.com/zephyrproject-rtos/zephyr/pull/46494#issuecomment-1154687751 depends on this one.

Reply:
Ok, if there is a dependency leave it and @galak can remove it later

Reply:
Right

Reply:
We need it for now.  Will look at cleaning the UART use of `label` up in the future, for now just add it.